### ISSUE TITLE
Deprecation warning popping up

### ISSUE THREAD

**Issue by mackeyja92** [hynek/argon2_cffi#16]
Getting this warning message using the library.

```
.../argon2/low_level.py:93: UserWarning: implicit cast to 'char *' from a different pointer type: will be forbidden in the future (check that the types are as you expect; use an explicit ffi.cast() if they are correct)
```

**Comment by hynek** [comment]
This has been fixed in master.  I'm currently waiting that virtualenv is fixed for Python 3.6 so I can merge #14 and release.

**Comment by hynek** [comment]
fixed in 16.3.0
